NavigationReactNative/src/ios/NVBarTitleView.m

```diff
@@ -0,0 +1,24 @@
+#import "NVBarTitleView.h"
+
+#import <React/UIView+React.h>
+
+@implementation NVBarTitleView
+
+- (void)didMoveToWindow {
+    [super didMoveToWindow];
+    
+    UIViewController *reactViewController = self.reactViewController;
+    if (reactViewController && self.window) {
```

Comment:
Don't think we need to check that they're not nil. The [similar code in NVSearchBarView](https://github.com/grahammendick/navigation/blob/cc2e1b2be84c39d3ce88843db85edea8d48929ca/NavigationReactNative/src/ios/NVSearchBarView.m#L100) doesn't check

Reply:
OK, this check is redundant. I'll remove it.